src/components/ProposalActions.js

```diff
@@ -143,20 +134,26 @@ function ProposalActions({
             `}
           >
             <Slider
-              value={progress}
-              onUpdate={setProgress}
+              value={
+                maxAvailable.gt(0) ? amount.div(maxAvailable).toNumber() : 0
+              }
+              onUpdate={newProgress =>
+                setInputValue(
+                  roundSlider(maxAvailable.multipliedBy(newProgress))
+                )
+              }
               css={`
                 padding-left: 0;
                 width: 100%;
               `}
             />
             <TextInput
               value={inputValue}
-              onChange={setAmount}
               type="number"
-              max={maxAvailable}
+              onChange={event => setInputValue(event.target.value)}
```

Comment:
Same here

Reply:
How can this function be putted in a callback? It doesn't depend on any other parameter.

Reply:
When it doesn't depend on any other parameter, it will just be the same function on every render.
In this cases we leave the array of dependencies empty :)